source/vibe/http/common.d

```diff
@@ -601,7 +601,7 @@ struct CookieValueMap {
 
 	inout(string)* opBinaryRight(string op)(string name) inout if(op == "in")
 	{
-		foreach(c; m_entries)
+		foreach(ref c; m_entries)
```

Comment:
Does your current code here compile with DMD 2.065? It gave an error about not being able to convert from `inout` to `inout` or similar, which is why the `static if` my fixing commit was still necessary.